Tribler/Test/Core/test_permid.py

```diff
@@ -29,3 +31,26 @@ def test_save_load_keypair_pubkey(self):
         self.assertIsInstance(loaded_key, EC)
 
         shutil.rmtree(unicode(test_base_dir), ignore_errors=True)
+
+    def test_save_load_keypair_pubkey_multichain(self):
+        permid.init()
+        key = permid.generate_keypair_multichain()
+
+        test_base_dir = mkdtemp(suffix="_tribler_test_session")
```

Comment:
When deriving from `TriblerCoreTest`, a temporary directory is already provided for each test (`self.session_base_dir`). We should use that one since that temp dir is always cleaned up properly when the tests are done, regardless of success or failure. If you agree, can you also change this in the other test in this file?